crates/zombie_actor/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(rustc_private)]
+extern crate backtrace;
```

Comment:
I think I added the `rustc_private` line to get the logger working but I didn't really understand why. Glad it's gone.